tools/docker/Dockerfile.ubuntu18.04

```diff
@@ -84,6 +86,11 @@ WORKDIR /tmp
 RUN cd gcc && ./build.sh && ./install_zlib.sh && rm -rf /tmp/gcc
 ENV PATH="/opt/occlum/build/bin:/usr/local/occlum/bin:$PATH"
 
+# Install glibc
+COPY toolchains/glibc /tmp/glibc
+WORKDIR /tmp
+RUN cd glibc && ./build.sh && rm -rf /tmp/glibc
```

Comment:
According to your proposal, the toolchain folder would look like this
```
clang/
gcc/
glibc/
golang/
java/
rust/
```
You see the issue here, right? `clang` and `gcc` are different C compilers; so they are exclusive. But `glibc` are libc. Adding a subfolder named `glibc` would immediately raise a question: what kinds of libc does `clang` or `gcc` use or they do not include a libc at all? To avoid any such confusion, I think we should rename `gcc/` to `musl-gcc/` and you can keep the name of the new folder as `glibc/` (which is a very precise description as it does contain GCC, only the libc). Furthermore, we can later remove the `clang` folder, which has been deprecated anyway.

Reply:
OK